### DISCUSSION TITLE
HELP! You have to change key -mapping of <tab> (LUA)

### DISCUSSION THREAD

**Discussion by Farzadin** [neoclide/coc.nvim#4012] [Q&A] [answered]
how can I satisfy this error my tab is no longer working after the update.

![coc-ask](https://user-images.githubusercontent.com/58443122/183221460-ce15e235-3655-4c72-8f0a-3aa90f2ef769.png)

**Comment by chemzqm** [comment]
See #3985

**Comment by Farzadin** [comment, marked as answer]
nvm I fix it by changing it to this 
```
vim.api.nvim_set_keymap("i", "<TAB>", "coc#pum#visible() ? coc#pum#next(1): '<TAB>'", {noremap = true, silent = true, expr = true})

```

**Comment by T-TRz879** [comment]
@thisiscyiran how can you deal with this problem,i did not has coc-config.lua

**Reply by Farzadin** [reply]
C:\Users\YourPcUername\AppData\Local\nvim) this is the defualt nvim config file directory. (For Windows only)

- Inside this directory make a New Folder and name it lua.
- then go inside this lua folder and make coc-config.lua file and then you can put your key mapping for coc plugin in that file.
- and in the end make sure to require that file in init.lua file which will be like this

```lua

require('coc-config')

```
